packages/web-app-admin-settings/src/composables/keyboardActions/useKeyboardTableMouseActions.ts

```diff
@@ -0,0 +1,77 @@
+import { onBeforeUnmount, onMounted, unref, Ref } from 'vue'
+import { eventBus } from 'web-pkg'
+import { KeyboardActions } from 'web-pkg/src/composables/keyboardActions'
+import { findIndex, find } from 'lodash-es'
+import { Resource } from 'web-client'
+
+export const useKeyboardTableMouseActions = (
+  keyActions: KeyboardActions,
+  paginatedResources: Ref<Resource[]>,
```

Comment:
`Resource` is only for files, we need `Group[] | User[] | Space[]` or maybe a generic interface? We probably only need the `id` prop, right?